schema/manifest_test.go

```diff
@@ -89,6 +89,23 @@ func TestManifest(t *testing.T) {
 			fail: true,
 		},
 
+		// expected failure: config.digest is not hex hashed format
+		{
+			manifest: `
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "config": {
+    "mediaType": "application/vnd.oci.image.config.v1+json",
+    "size": 1470,
+    "digest": "sha256:c86f7763873b6c0aae22d9h3bab59b4f5debbed6685761b5951584f6efb0633b"
```

Comment:
Could you make this less subtle?

There is an `h` in the middle for anyone not keeping count.